src/components/layouts/collection-page-layout.tsx

```diff
@@ -1094,4 +1204,92 @@ const transformSanityTopics = (topics: any) => {
   return topics.items
 }
 
+const DialogButton = ({title, buttonStyles, buttonText, children}: any) => {
+  const [showDialog, setShowDialog] = React.useState(false)
+
+  const openDialog = () => {
+    setShowDialog(true)
+  }
+  const closeDialog = () => {
+    setShowDialog(false)
+  }
+
+  return (
+    <>
+      <button className={buttonStyles && buttonStyles} onClick={openDialog}>
+        <LockIcon className="h-4 w-4 mr-2" /> {buttonText}
+      </button>
+      <DialogOverlay
+        isOpen={showDialog}
+        onDismiss={closeDialog}
+        className="bg-black/50 backdrop-blur-sm flex justify-center items-center p-4 xs:px-6 xs:py-8 z-50"
+      >
+        <DialogContent
+          aria-label="sign in to bookmark"
+          className="bg-white dark:bg-gray-900 shadow-lg rounded-lg max-w-screen-sm border dark:border-gray-800 relative p-3 xs:p-6 w-112"
+        >
+          <div className="w-full flex flex-col">
+            {title && (
+              <h4 className="text-lg sm:text-xl mb-4 font-semibold text-center px-4 leading-tight">
+                {title}
+              </h4>
+            )}
+            {children && children}
+          </div>
+          <div className="block absolute top-0 right-0 pt-3 pr-3">
+            <button
+              onClick={closeDialog}
+              type="button"
+              className={`text-gray-500 dark:text-gray-400 hover:bg-blue-100 hover:text-blue-600 dark:hover:text-blue-300 dark:hover:bg-gray-700 p-1 focus:shadow-outline-blue transition-all rounded-full hover:scale-110 ease-in-out duration-200`}
+              aria-label="Close"
+            >
+              <span className="sr-only">close feedback dialog</span>
+              {/* prettier-ignore */}
+              <svg className="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor"><path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M6 18L18 6M6 6l12 12" /></svg>
+            </button>
+          </div>
+        </DialogContent>
+      </DialogOverlay>
+    </>
+  )
+}
+
+const PlanFeatures = () => {
+  const DEFAULT_FEATURES = [
```

Comment:
suggestion [single source]: this data is also used in the "pick a plan" component with a different presentation. I think it makes sense for them both to say the same thing and use the same set of features.

https://github.com/eggheadio/egghead-next/blob/2b7c352b4ccb2b8d641b0ba45274e3fed6057c72/src/components/pricing/select-plan/index.tsx

The `SelectPlan` component is a more complex presentation, but it's the same data and seeing it copied here made me think we might just want to have the data aspect in one place.

Reply:
![image](https://user-images.githubusercontent.com/518406/133489785-f24ce875-62e6-4c30-9ced-2ed3046a4c63.png)
Here is what the modal looks like using the pricing features list